styles/footer.css

```diff
@@ -0,0 +1,89 @@
+.mainFont {
+  font-family: "FjallaOne";
+  color: rgba(255, 255, 255, 0.6);
+  text-transform: uppercase;
+  line-height: 1.4;
+  text-align: center;
+}
+.arrow {
+  position: absolute;
+  align-self: center;
+  margin-top: 8px;
+  width: 32px;
+  height: 47px;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  z-index: 3;
+}
+.arrow:hover {
+  cursor: pointer;
+}
+.MyriadProFont {
+  font-size: 13px;
+  font-family: "Myriad Pro";
+}
+.OxygenFont {
+  font-family: "Oxygen";
+  font-size: 13px;
+  color: #979797;
+  text-decoration: underline;
+}
+footer {
+  width: 100%;
+  background: url(../images/topbg.jpg);
+  background-size: 100% 100%;
+  font-size: 14px;
+}
+.footer__content {
+  margin: auto;
+  max-width: 1170px;
+  display: flex;
+  flex-flow: column nowrap;
+  font-family: "OpenSans";
+}
+.footer__topPart {
+  display: flex;
+  border-bottom: 1px solid #f0f0f0;
+  padding: 45px 0 30px 0;
+  flex-flow: row wrap;
+}
+.footer__topPart-item {
+  color: #b297c4;
+  flex: 200px 1 1;
+  padding: 0 40px 5px 0;
+  font-weight: bold;
+}
+.footer__topPart-item_text {
+  color: #979797;
+  font-weight: 100;
+  margin-top: 5px;
+  display: block;
+}
+a {
+  color: #979797;
+  text-decoration: none;
+}
+.footer__bottomPart {
+  display: flex;
+  font-weight: bold;
+  align-items: center;
+  justify-content: flex-start;
+}
+.footer__bottomPart-item {
+  color: #b297c4;
+  margin: 40px 160px 65px 0;
+}
+.footer__bottomPart a {
```

Comment:
You should not create css rules for selectors in bem. Please, add element class for this selectors